projects/ad6676evb: Add README files (#1708)
analogdevicesinc/hdl: projects/ad6676evb/vc707/README.md

# AD6676-EVB/VC707 HDL Project

## Building the project

> The parameters configurable through the `make` command, can be found below, as well as in the **system_project.tcl** file; it contains the default configuration.

Reviewer: ```suggestion
The parameter configurable through the `make` command, can be found below, as well as in the **system_project.tcl** file; it contains the default configuration.
```

```
cd projects/ad6676evb/vc707
make
```

All of the RX/TX link modes can be found in the [AD6676 data sheet](https://www.analog.com/media/en/technical-documentation/data-sheets/AD6676.pdf). We offer support for only a few of them.

> If other configurations are desired, then the parameters from the HDL project (see below) need to be changed, as well as the Linux/no-OS project configurations.

Reviewer: ```suggestion
If other configurations are desired, then the parameter from the HDL project (see below) needs to be changed, as well as the Linux/no-OS project configurations.
```

> The overwritable parameters from the environment:

Reviewer: ```suggestion
The overwritable parameter from the environment:
```
